src/stage-release.sh

```diff
@@ -26,13 +26,17 @@ if [ $# -eq 0 ]; then
 fi
 
 DEST_PATH=$1
+DEST_PATH="$(cd "$DEST_PATH" && pwd)"
 
 pushd $(dirname "$0")
 PULSAR_PATH=$(git rev-parse --show-toplevel)
 VERSION=`./get-project-version.py`
 popd
 
-cp $PULSAR_PATH/target/apache-pulsar-$VERSION-src.tar.gz $DEST_PATH
+pushd "$(dirname "$0")/.."
```

Comment:
RM will still use `$PULSAR_PATH/src/stage-release.sh .` so it's an internal impl change which doesn't cause any external visible change.